src/main.cpp

```diff
@@ -300,7 +499,7 @@ int main(int argc, char *argv[]) {
 
   for (string const& kernel_name : kernel_list) {
     exec_time = exec_time + dev_mgr.execute_kernelNA(*(dev_mgr.getKernelbyName(0, "ocl_Kernel", kernel_name)),
-                                                     dev_mgr.get_queue(0, 0), range_start, global_range, local_range);
```

Comment:
Why did you delete the whitespace here? It is harder to read (at least for me) in this new version.

Reply:
It seems like changing between VS2017/VS2015 and VS Code has some drawbacks regarding auto indentations, whitespaces and so on.